baseline.xml

```diff
@@ -27,7 +27,17 @@
       <code><![CDATA[$this->getReference(VocabularyFixtures::MANAGED)]]></code>
     </ArgumentTypeCoercion>
   </file>
-  <file src="src/Services/Feeds/Mapper/Source/FeedItemSource.php">
+  <file src="src/MessageHandler/EventHandler.php">
+    <PossiblyNullArgument>
+      <code><![CDATA[$entity->getId()]]></code>
+    </PossiblyNullArgument>
+  </file>
```

Comment:
I don't understand why we need this and I think it's dangerous to ignore this error.

Reply:
Fixed